content/governance.md

```diff
@@ -0,0 +1,175 @@
+
+New members may be proposed by any existing member by email to [prometheus-team][team]. It is highly desired to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
+
+Team members are added to the [GitHub organization][gh] as _Owner_. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately.
+
+The current team members are:
+
+* Ben Kochie
+* Björn Rabenstein
+* Brian Brazil
+* Conor Broderick
+* Fabian Reinartz
+* Frederic Branczyk
+* Goutham Veeramachaneni
+* Johannes Ziemke
+* Julius Volz
+* Matt Layher
+* Matthias Rampke
+* Max Inden
+* Richard Hartmann
+* Steve Durrheimer
+* Stuart Nelson
+* Tobias Schmidt
+* Tom Wilkie
+
+### Maintainers
+
+Maintainers lead one or more project(s) or parts thereof and serve as a point of conflict resolution amongst the contributors to this project. Ideally, maintainers are also team members, but exceptions are possible for suitable maintainers that, for whatever reason, are not or not yet team members.
+
+Changes in maintainership have to be announced on the [developers mailing list][devs]. They are decided by [lazy consensus](#consensus) and formalized by changing the `MAINTAINERS.md` file of the respective repository.
+
+Maintainers are granted commit rights to all projects in the [GitHub organization][gh].
+
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers who wish to resign are encouraged to propose another team member to take over the project.
+
+A project may have multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+Any maintainership change is performed by changing the project's MAINTAINERS.md on the main branch of the project.
+
+### Technical decisions
+
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [lazy consensus](#consensus) is assumed.
+
+Technical decisions that span multiple parts of the Prometheus Project can be discussed and made on the [Prometheus developer mailing list][devs]. Decisions are usually made by [lazy consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
```

Comment:
Perhaps change the first "can" to "should"?

How about starting a new paragraph after the first sentence? The rest of the paragraph also applies to single-project decisions (as even a decision local to a project might be disputed).

Reply:
yes, should be should, changed.

split the paragraph, but then merged the two sentences about technical decisions into one (keeping my own love for single-sentence paragraphs in check).